example/mysite/docker_settings.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
```

Comment:
This whole file could do with some documentation, but I am planning to do that in a separate PR where I update all the documentation for the project and use this settings file as an example of some best practices.